### PR TITLE
Limit GA data fetch to 1 year

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Review by duallain — APPROVED** [review]
makes a ton of sense to move from forever to 1 year, especially in the context of it sometimes failing, sometimes working.  I'm not the best person to review the djangoy bits of this pr, but the idea seems great from the pov of snitches firing fairly regularly.

File: kitsune/questions/models.py [hunk 1]
```diff
@@ -708,9 +709,8 @@ def reload_from_analytics(cls, verbose=False):
         """Update the stats from Google Analytics."""
         from kitsune.sumo import googleanalytics
 
-        counts = googleanalytics.pageviews_by_question(
-            settings.GA_START_DATE, date.today(), verbose=verbose
-        )
+        one_year_ago = datetime.now() - timedelta(days=365)
```
Comment (LeoMcA) [inline kitsune/questions/models.py:712]: This'll be a `datetime`, whereas previously the first argument to `pageviews_by_question` was a `date`, could this pose a problem?

**Review by LeoMcA — APPROVED** [review]
looks good, one query

File: kitsune/questions/models.py (hunk 1 above)
Comment (akatsoulas) [reply kitsune/questions/models.py:712]: it will just add the time portion too but that's a good flag. I am updating this back to date for consistency.

**Pull request merged**